apps/test/unit/templates/studioHomepages/SectionsTableTest.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import i18n from '@cdo/locale';
+import {joinedSections} from './homepagesTestData';
+import SectionsTable from '@cdo/apps/templates/studioHomepages/SectionsTable';
+
+describe('SectionsTable', () => {
+
+  it('shows column headers for students with teacher-managed accounts', () => {
+    const wrapper = shallow(
+      <SectionsTable
+        sections={joinedSections}
+        isRtl={false}
+        isTeacher={false}
+        canLeave={false}
+      />
+    );
+    expect(wrapper).to.containMatchingElement(
```

Comment:
There's a lot of repetition in the structure of the column headers here - I wonder if a loop would actually read better, because it would be obvious what difference was being asserted.  Something like this:
```jsx
[
  i18n.section(),
  i18n.course(),
  i18n.teacher(),
  i18n.sectionCode(),
].forEach((headerText) => {
  expect(wrapper).to.containMatchingElement(
    <td>
      <div>
        {headerText}
      </div>
    </td>
  )
});
```